src/main/java/org/traccar/protocol/Gl200TextProtocolDecoder.java

```diff
@@ -51,6 +52,39 @@ public class Gl200TextProtocolDecoder extends BaseProtocolDecoder {
 
     private final DateFormat dateFormat;
 
+    private static final HashMap<String, String> DEVICE_MODELS = new HashMap<String, String>() {{
```

Comment:
Statics should be above instance variables.

Reply:
ok, but the patterns are not.

Reply:
They used to be, but it made sense to make an except. Note again that the plan is to get rid of patterns eventually.